### PR TITLE
Feature encrypt strings

### PR DESCRIPTION
**Pull request by bezmen**
EncryptionStrings

### CODE REVIEW

File: encoder.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            
+            StreamReader reader = new StreamReader("text.txt");
+            Console.WriteLine(reader.ReadLine());
+            Console.WriteLine(reader.ReadLine());
+            reader.Close();
+            Console.ReadKey();
+        }
+        string Encryption(ref string str)
+        {
+            char a;
```
Comment (cobraalloc) [inline encoder.cs:32]: _**Please choose names according to their purpose.**
Don't declare many of the same functional variables, declare them as class field and pass it when the function is called, as an argument._

File: encoder.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            for (int i = 0; i < str.Length; i++)
+            {
+                a = str[i];
+                a <<= 1;
+                outStr += a;
+            }
+            str = outStr;
+            return str;
+        }
+        string Decript(ref string str)
```
Comment (cobraalloc) [inline encoder.cs:45]: Decr**Y**pt

Comment (cobraalloc) [reply encoder.cs:45]: - [x] This is not unicode, but character conversion to hex. 
- [ ] In project file I create a class for this unit (search this in task list with tagged '//TODO'), please paste your code into this unit and resend me pr.

File: encoder.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                a = str[i];
+                a >>= 1;
+                outStr += a;
+            }
+            str = outStr;
+            return str;
+        }
+    }
+}
+//lampw
```
Comment (bezmen) [inline encoder.cs:60]: - [x] Learn what is unicode code

**Pull request closed without merging**